Configuration/TypoScript/Plugins/kitodo.typoscript

```diff
@@ -3,17 +3,17 @@
 # ------------------------------------
 
 # map set[mets] --> tx_dlf[id]
-[like(traverse(request.getQueryParams(), 'set/mets'), 'http*')]
+[request?.getQueryParams()?['set']?['mets'] =~ /http.*/]
```

Comment:
```suggestion
like(request?.getQueryParams()?['set']?['mets'], 'http*')]
```

Reply:
I am afraid that more tests are needed to find a solution which does not break different use cases.